CONTRIBUTING.md

```diff
@@ -0,0 +1,89 @@
+# Contributiung to Turf.dart
+
+Welcome and thank you for deciding to contribute to the project!
+
+Here is how cooperation works perfectly at [Turf Dart](https://github.com/dartclub/turf_dart)
+#### Table of Contents
+  - [Code of Conduct](#code-of-conduct)
+  - [Get started](#get-started)
+  - [Structure of modules](#structure-of-modules)
+  - [Implementation Process](#implementation-process)
+  - [Documentation](#documentation)
+
+## Code of conduct
+By participating, you are expected to uphold international human rights and fundamental freedoms!
+To put it simply, be kind to each other. 
+
+## Get started
+- Get the [tools](https://dart.dev/tools)
```

Comment:
I think, "Install the dart SDK" https://dart.dev/tools/sdk would be a little bit more clear